gothemis/cell/cell.go

```diff
@@ -190,6 +190,8 @@ var (
 	ErrMissingToken      = errors.NewWithCode(errors.InvalidParameter, "authentication token is required in Token Protect mode")
 	ErrMissingContext    = errors.NewWithCode(errors.InvalidParameter, "associated context is required in Context Imprint mode")
 	ErrOutOfMemory       = errors.NewWithCode(errors.NoMemory, "Secure Cell cannot allocate enough memory")
+	// Deprecated: Since 0.13.3. Use ErrOutOfMemory instead.
+	ErrOverflow          = ErrOutOfMemory
```

Comment:
This adds a whole bunch of new API so I believe this will go into the next release, 0.14.0. Even it there was no new API, it's not nice to deprecate stuff in minor releases. Please change the comments here and there to refer to 0.14.0.